context/app/static/js/components/Detail/StatusIcon.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import LensIcon from '@material-ui/icons/Lens';
+
+const ColoredStatusIcon = styled(LensIcon)`
+  color: ${(props) => props.theme.palette[props.$iconColor].main};
+  font-size: 16px;
+  margin-right: 3px;
+`;
+
+function getColor(status) {
+  if (['NEW', 'REOPENED', 'QA', 'LOCKED', 'PROCESSING', 'HOLD'].includes(status)) {
+    return 'info';
+  }
+
+  if (['INVALID', 'ERROR'].includes(status)) {
+    return 'error';
+  }
+
+  if (['UNPUBLISHED', 'DEPRECATED'].includes(status)) {
+    return 'warning';
+  }
+
+  if (status === 'PUBLISHED') {
+    return 'success';
+  }
+
+  return null;
```

Comment:
```suggestion
  return 'error';
```
perhaps? It is an error, and not having to deal with the null downstream makes our lives easier.